uniffi/src/interface/mod.rs

```diff
@@ -59,6 +59,8 @@ pub mod types;
 pub use types::TypeReference;
 use types::TypeResolver;
 
+const GENERIC_RUST_ERROR: &str = "GenericRustError";
```

Comment:
Aaaah using this to be the name of the error that non-error functions take as there out `err`. It was the lazy thing to do...

Reply:
Does this mean non-throwing functions still return a `Result<T, GenericRustError>` or?

Reply:
Ya! And I can get rid of this, but it can be used as a bucket for all 'Unknown' kind of errors (a panic comes to mind, but that'll be a part of #32). Also kinda allows me to go `call_with_result` every time.. 

I'm not a big fan and it feels very hardcoded-y and now I take a second look at it, I'll get rid of it, and have the effort to deal with #32 deal with generic issues that might happen. 😄